src/python/pants/core/goals/tailor.py

```diff
@@ -285,6 +285,36 @@ def register_options(cls, register):
             f"to the one it replaces (see {doc_url('macros')}).",
         )
 
+        register(
+            "--ignore-paths",
+            advanced=True,
+            type=list,
+            member_type=str,
+            help=(
+                "Do not edit or create BUILD files at these paths.\n\n"
+                "Can use literal file names and/or globs, e.g. "
+                "`['project/BUILD, 'ignore_me/**']`.\n\n"
+                "This augments the option `[GLOBAL].build_ignore`, which tells Pants to also not "
+                "_read_ BUILD files at certain paths. In contrast, this option only tells Pants to "
+                "not edit/create BUILD files at the specified paths."
+            ),
+        )
+        register(
+            "--ignore-adding-targets",
```

Comment:
I want to make clear that this is about adding new targets. When we unify `tailor` and `update-build-files`, this option will not impact the fixers like Black and deprecation fixers. That's too difficult to pull off.

One concern I have is that we should add the option `--add-targets` when that unification happens so that you can have Pants fix deprecations & run Black w/o having to add new targets. Not sure if `--add-targets` and `--ignore-adding-targets` is too similar.

Other ideas welcomed.